orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/MessageHandler.kt

```diff
@@ -83,4 +83,23 @@ interface MessageHandler<M : Message> : (Message) -> Unit {
     } catch(e: ExecutionNotFoundException) {
       queue.push(InvalidExecutionId(this))
     }
+
+  fun Stage<*>.startNext() {
```

Comment:
Just moved this method from `CompleteStageHandler` to somewhere it could be shared with the new `SkipStageHandler`.